centreon-open-tickets/www/modules/centreon-open-tickets/providers/Ivanti/IvantiProvider.class.php

```
@@ -0,0 +1,838 @@
+        $tpl->assign('service_selected', $service_problems);
+        // assign submitted values from the widget to the template
+        $this->assignSubmittedValues($tpl);
+
+        $ticketArguments = $extraTicketArguments;
+        if (isset($this->rule_data['clones']['mappingTicket'])) {
+            // for each ticket argument in the rule form, we retrieve its value
+            foreach ($this->rule_data['clones']['mappingTicket'] as $value) {
+                $tpl->assign('string', $value['Value']);
+                $resultString = $tpl->fetch('eval.ihtml');
+                if ($resultString == '') {
+                    $resultstring = null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resultstring'. (undefined)

(UnusedLocalVariable)